.github/workflows/tests.yaml

```
@@ -116,7 +117,8 @@ jobs:
           ${{ runner.os }}-go-
     - run: go version
     - run: go mod download # Not required, used to segregate module download vs test times
-    - run: (cd /tmp && go get -u github.com/onsi/ginkgo/ginkgo@v1.16.5)
+    - run: (cd /tmp && go get -u github.com/onsi/ginkgo/v2 && go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@latest)
```

We no longer need to keep backwards compatibility at this point, we can just use install
```suggestion
    - run: (cd /tmp && go install -mod=mod github.com/onsi/ginkgo/v2/ginkgo@latest)
```